ci/docker_cache.py

```diff
@@ -34,11 +32,11 @@
 import build as build_util
 from util import retry
 
-DOCKERHUB_LOGIN_NUM_RETRIES = 5
-DOCKERHUB_RETRY_SECONDS = 5
-DOCKER_CACHE_NUM_RETRIES = 3
-DOCKER_CACHE_TIMEOUT_MINS = 15
-PARALLEL_BUILDS = 10
+DOCKERHUB_LOGIN_NUM_RETRIES =   int(os.environ.get('DOCKER_CACHE_DOCKERHUB_LOGIN_NUM_RETRIES', '5'))
+DOCKERHUB_RETRY_SECONDS =       int(os.environ.get('DOCKER_CACHE_DOCKERHUB_RETRY_SECONDS', '5'))
+DOCKER_CACHE_NUM_RETRIES =      int(os.environ.get('DOCKER_CACHE_DOCKER_CACHE_NUM_RETRIES', '3'))
+DOCKER_CACHE_TIMEOUT_MINS =     int(os.environ.get('DOCKER_CACHE_DOCKER_CACHE_TIMEOUT_MINS', '15'))
+PARALLEL_BUILDS =               int(os.environ.get('DOCKER_CACHE_PARALLEL_BUILDS', '10'))
```

Comment:
Jenkins environment variables are not versioned so far and thus can be changed without notice. For configuration like this, I'd prefer if we would have it in versioned configuration if you don't mind.